test/functional/apps/visualize/_embedding_chart.js

```diff
@@ -1,187 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Any modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
- */
-
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import expect from '@osd/expect';
-
-export default function ({ getService, getPageObjects }) {
-  const filterBar = getService('filterBar');
-  const log = getService('log');
-  const renderable = getService('renderable');
-  const embedding = getService('embedding');
-  const PageObjects = getPageObjects([
-    'visualize',
-    'visEditor',
-    'visChart',
-    'header',
-    'timePicker',
-  ]);
-
-  describe('embedding', () => {
-    describe('a data table', () => {
```

Comment:
huh, interesting that it only tested embedding a data table. Has this test also been migrated to the functional test repo, even though it's outside of testing the table vis itself?

Reply:
It hasn't yet. But I am working on it. 
Currently, all the table functions and tests on visualization page are re-produced. There are two things left: 1) embedding a table 2) add table to dashboard page

Reply:
cool, yeah, as long as we're tracking tests that still require migration somewhere.